src/src/DataStructs/NodeStruct.h

```diff
@@ -30,6 +30,7 @@ struct NodeStruct
   uint16_t  build;
   byte      age;
   byte      nodeType;
+  uint16_t  webgui_portnumber;
```

Comment:
Make sure to set this to 0 explicitly when constructing it.
Older nodes may not send this data.

Reply:
missed that